master/internal/api_project.go

```diff
@@ -330,6 +463,67 @@ func (a *apiServer) getProjectColumnsByID(
 	}, nil
 }
 
+func (a *apiServer) getProjectRunColumnsByID(
+	ctx context.Context, id int32, curUser model.User,
+) (*apiv1.GetProjectColumnsResponse, error) {
+	p, err := a.GetProjectByID(ctx, id, curUser)
+	if err != nil {
+		return nil, err
+	}
+
+	columns := defaultRunsTableColumns
+
+	hyperparameters := []struct {
+		WorkspaceID int
+		Hparam      string
+		Type        string
+	}{}
+
+	// get all runs in project
```

Comment:
nit: comments should be sentences starting with a capital letter and ending with a period.